lib/core/engine.js

```diff
@@ -256,8 +246,8 @@ class Engine {
         // because the contractsManager config is corrupted after a deploy
         if (fileType === 'contract' || fileType === 'config') {
           self.config.reloadConfig();
-          self.deployManager.deployContracts(function () {
-          });
+
+          self.events.request('deploy:contracts', () => {});
```

Comment:
Can you make the callback optional?

Reply:
next PR (since I might need to update much more lines and remove unnncessary callbacks)